### ISSUE TITLE
Bugs: router cannot access anything on br-lan

### ISSUE THREAD

**Issue by pozzo-balbi** [openNDS/openNDS#526]
Hi,

I have the problem that I am running OpenNDS on the main router, that is also used to monitor the network. If OpenNDS is running my router cannot ping or access any device on br-lan. 

Also adding the trustedmacaddress, since it is not a client receiving an IP via Dhcp, as explained here https://github.com/openNDS/openNDS/issues/510. 

Since I am not yet familiar with nftables, I can't fix it yet myself, but as a simple question, why is the router running OpenNDS prevented from scanning his own br-lan network?

```
uci show opennds
opennds.@opennds[0]=opennds
opennds.@opennds[0].enabled='1'
opennds.@opennds[0].gatewayinterface='br-lan'
opennds.@opennds[0].maxclients='1500'
opennds.@opennds[0].upload_bucket_ration='0'
opennds.@opennds[0].download_bucket_ration='0'
opennds.@opennds[0].users_to_router='allow udp port 53' 'allow tcp port 53' 'allow udp port 67' 'allow udp port 123' 'allow tcp port 80' 'allow tcp port 443'
opennds.@opennds[0].preauthenticated_users='allow tcp port 80 to 10.10.0.1' 'allow tcp port 80 to 192.168.10.1'
opennds.@opennds[0].faskey='123456'
opennds.@opennds[0].gatewayname='WiFi'
opennds.@opennds[0].max_log_entries='100000'
opennds.@opennds[0].login_option_enabled='3'
opennds.@opennds[0].themespec_path='/usr/lib/opennds/theme_voucher.sh'
opennds.@opennds[0].download_unrestricted_bursting='0'
opennds.@opennds[0].upload_unrestricted_bursting='0'
opennds.@opennds[0].sessiontimeout='0'
opennds.@opennds[0].authidletimeout='600000'
opennds.@opennds[0].uploadrate='0'
opennds.@opennds[0].downloadrate='0'
opennds.@opennds[0].log_mountpoint='/mnt/sda1'
opennds.@opennds[0].trustedmac='xxxxxxx`
```

**Comment by bluewavenet** [comment]
@pozzo-balbi 

> why is the router running OpenNDS prevented from scanning his own br-lan network?

It's not, and this cannot be replicated here.
It is most likely either an error in your configuration or something set in the OpenWrt firewall.

Why are you allowing preauthenticated access to 10.10.0.1 and 192.168.10.1?

**_What versions of OpenWrt and openNDS are you using here?_**

PS:

```
opennds.@opennds[0].upload_bucket_ration='0'
opennds.@opennds[0].download_bucket_ration='0'
```

These should be `xxx_bucket_ratio`, but will be ignored anyway as you have set rates to zero.

**Comment by bluewavenet** [comment]
@pozzo-balbi 

> It's not, and this cannot be replicated here.

Apologies, I misread your original post.

Versions of v10 before v10.2.0 block inbound traffic to the router on lan on unspecified random ports for security reasons and have no means of configuring to allow. An option to define a ruleset for this is available in v10.2.0. 

v10.2.0 is currently in beta testing.
